### ISSUE TITLE
Fluent Bit in Production: Add Qualtrics logo

### ISSUE THREAD

**Issue by meehanman** [fluent/fluent-bit#6443]
[Qualtrics](https://www.qualtrics.com/) has been using Fluent-bit across our datacenters for the past year for our logging infrastructure if you'd like to also add our logo to the Fluent bit in Production section. The small size, and high performance has noticeably reduced the costs associated with log collection. Kudos to the Fluentd team! 🎉 

![image](https://user-images.githubusercontent.com/4993167/203552735-91030a6c-ef39-4c33-9cef-231223e5577a.png)

## Usage Guidelines

You can find copies of our logo and branding in the [Qualtrics Brand Book](https://www.qualtrics.com/brand-book/)

![image](https://user-images.githubusercontent.com/4993167/203552912-06be82a9-ff0d-405d-94a4-88ee66f05349.png)

TLDR; 

X = SAFE AREA

The Qualtrics logo must always have a safe area free from any other elements.

DO NOT

- Fill logo with a texture or image
- Skew or distort logo in any way
- Use a 3D version of the logo unless creating a physical version. (i.e. signage etc)
- Do not use the logo in any other color except the approved color version above
- Do not change the proportions or elements of the lockup
- Do not add or attach any taglines as a lockup

**Comment by agup006** [comment]
Awesome thanks @meehanman ! Great to see this returning from the US Thanksgiving holiday :), will look to update fluentbit.io in the next couple weeks

**Comment by agup006** [comment]
Thanks again @meehanman , would love to have a use case blog with the CNCF or other venue if you are interested! I've finished getting the logo on the homepage!
